sources/packages/api/src/services/student/student.service.ts

```diff
@@ -164,7 +164,7 @@ export class StudentService extends RecordDataModelService<Student> {
       phone: studentInfo.phone,
     };
     student.user = user;
-
+    student.sinConsent = true;
```

Comment:
The `sinConsent` was added to the `StudentInfo` but is assigned here with a hard-coded value and it is also assigned in other places with a hard-coded value. I was not able to follow the idea here. Can you please elaborate?

Reply:
Fixed.